accelerator/apis.c

```diff
@@ -1,5 +1,28 @@
 #include "apis.h"
 
+// TODO: Generate actual pre shared keys
+mam_psk_t const psk = {
```

Comment:
Marked as "static", otherwise the symbol `psk` will be exposed to other compilation units.